flink-table/flink-sql-client/src/test/resources/sql/set.q

```diff
@@ -0,0 +1,80 @@
+# set.q - SET/RESET configuration
+# you can run CliClientITCase to test this sql script
+# ==========================================================================
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# validation test
+set execution.parallelism = 10a;
```

Comment:
add some cases to override the default properties and then call `set` or `reset`

Reply:
I think the following `SET table.sql-dialect=hive;` has already cover this?